src/Compilers/Core/Portable/CommandLine/CommonCompiler.cs

```diff
@@ -244,6 +246,233 @@ internal SourceText TryReadFileContent(CommandLineSourceFile file, IList<Diagnos
             }
         }
 
+        internal static ImmutableArray<ImmutableDictionary<string, ReportDiagnostic>> GetAnalyzerConfigOptions(
```

Comment:
>GetAnalyzerConfigOptions [](start = 86, length = 24)

I lack context to understand what this method is supposed to do. What are the arguments? Can you add a doc comment please?